components/CommentCount/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+class CommentCount extends React.Component {
+  formatString(count) {
+    if(count === 1) {
+      return `${count} comment`
```

Comment:
Feels as though we are reimplementing gettext plurals, https://www.gnu.org/savannah-checkouts/gnu/gettext/manual/html_node/Plural-forms.html

I'd imagine there is a similar library for JS.